File: Java/Java-Kompendium  - Professionell Java programmieren lernen/uebung3a.java
/*
1.  Erstellen Sie ein Programm, das ein Array aus sechs String-Variablen erzeugt.
    Dieses soll die Vornamen der Teilnehmer an einem Sprachkurs enthalten.
    Geben Sie das gesamte Array sowie den ersten und den letzten Namen der Liste aus.
    Ändern Sie danach einen beliebigen Eintrag und geben Sie das Array erneut aus.

Neumann, Markus. Java: Kompendium: Professionell Java programmieren lernen (German Edition) (S.80). Kindle-Version. 
*/

import java.util.*;

public class uebung3a {
    public static void main(String[] args)
    {
        String[] kurs = {"Paul","Melanie","Nadine","Thorsten","Kevin","Daniel"};

        System.out.println("Gesamter Kurs: " + Arrays.toString(kurs));
        System.out.println("Erster Eintrag: " + kurs[0]);
        System.out.println("Letzter Eintrag: " + kurs[5]);

        kurs[2] = "Simon";
        System.out.println("Neue Kursliste: " + Arrays.toString(kurs));
    }
}

File: C/C - Programmieren für Einsteiger/uebung3a.c
/*
1.  Erstellen Sie ein Programm mit einem Array, das fünf Fließkommazahlen enthält.
    Geben Sie die Werte einzeln aus.

Neumann, Markus. C Programmieren: für Einsteiger: Der leichte Weg zum C-Experten (Einfach Programmieren lernen 8) (German Edition) (S.72). Kindle-Version.
*/

#include <stdio.h>

void main()
{
    float x[] = {1.0, 2.1, 3.2, 4.3, 5.4};

    printf("Zahl 1: %f\n", x[0]);
    printf("Zahl 2: %f\n", x[1]);
    printf("Zahl 3: %f\n", x[2]);
    printf("Zahl 4: %f\n", x[3]);
    printf("Zahl 5: %f\n", x[4]);
}